apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -60,26 +51,17 @@ const nationalityData = [
   },
 ];
 
-const maxAge = 110;
-
 const initialFormState = {
   id: null,
   gender: 'male',
   name: null,
   lastName: null,
   passportId: null,
   nationality: null,
-  date: {
-    day: '',
-    month: '',
-    year: '',
-  },
-  inputErrors: {
-    day: '',
-    year: '',
-  },
+  dateOfBirth: null,
   bags: null,
   passengerCount: 1,
+  getDateRequest: false,
```

Comment:
This name is little confuse me if something starts with get I expect that is a function.

Reply:
Hmm what about `toggleDateRequest` or `dateRequestOn`?

Reply:
Changed to `isDateRequested`